webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala

```diff
@@ -632,39 +624,27 @@ object DisplayNode extends Loggable {
   private def htmlId_#(jsId: JsNodeId, prefix: String): String = "#" + prefix + jsId.toString
 
   // Display the role of the node
-  private def displayServerRole(sm: FullInventory, inventoryStatus: InventoryStatus): NodeSeq = {
-    val nodeId = sm.node.main.id
-    inventoryStatus match {
+  private def displayServerRole(nodeFact: NodeFact): NodeSeq = {
+    nodeFact.rudderSettings.status match {
       case AcceptedInventory =>
-        val nodeInfoBox = nodeInfoService.getNodeInfo(nodeId).either.runNow
```

Comment:
this is not relevant anymore. Plus, avoid more queries when we already have the data